### DISCUSSION TITLE
Automatically disable "Place behind graphs" when importing images

### DISCUSSION THREAD

**Discussion by Heavenira** [DesModder/DesModder#855] [Ideas]
Just a suggestion! Not crucial. I manually import images into graphs quite often, bothers me to right-click the toggle.

**Comment by jared-hughes** [comment]
Maybe this could be treated as a general "apply action to all stuff."

- collapse all folders
- unplace all images behind graph
- mark all valid expressions as GLesmos
- etc. (any more ideas?)

**Reply by ronwnor** [reply]
omg yes auto-glesmify would be sick I'd love it 

for more ideas - I noticed a question popping form time to time in desmo server about how to set multiple expressions' color simultaneously. tbh I don't think it really belongs here, but maybe a per-folder style tray could fit in folder-tools?

anyway yes auto-gelsmify
